File: app/src/main/java/aipova/githubsearch/dao/RepoSearchResult.java
package aipova.githubsearch.dao;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by aipova on 09.02.17.
 */

public class RepoSearchResult {
    @SerializedName("total_count")
    @Expose
    private Integer totalCount;
    @SerializedName("incomplete_results")
    @Expose
    private Boolean incompleteResults;
    @SerializedName("items")
    @Expose
    private List<RepoModel> items = null;

    public Integer getTotalCount() {
        return totalCount;
    }

    public void setTotalCount(Integer totalCount) {
        this.totalCount = totalCount;
    }

    public Boolean getIncompleteResults() {
        return incompleteResults;
    }

    public void setIncompleteResults(Boolean incompleteResults) {
        this.incompleteResults = incompleteResults;
    }

    public List<RepoModel> getItems() {
        return items;
    }

    public void setItems(List<RepoModel> items) {
        this.items = items;
    }

}

File: app/src/main/java/aipova/githubsearch/dao/RepoSearchResult.kt
package aipova.githubsearch.dao

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by aipova on 09.02.17.
 */

class RepoSearchResult {
    @SerializedName("total_count")
    @Expose
    var totalCount: Int? = null
    @SerializedName("incomplete_results")
    @Expose
    var incompleteResults: Boolean? = null
    @SerializedName("items")
    @Expose
    var items: List<RepoModel>? = null

}
